sdk/search/Azure.Search.Documents/tests/DocumentOperations/SearchTests.cs

```diff
@@ -1008,6 +1008,30 @@ await foreach (Page<SearchResult<Hotel>> page in response.Value.GetResultsAsync(
                 ids);
         }
 
+        [Test]
+        public void SearchOptionsCanBeCopied()
+        {
+            SearchOptions source = new();
+
+            source.Facets = new List<string> { "facet1", "facet2" };
+            source.Filter = "searchFilter";
+            source.Select = null;
```

Comment:
Should this test not be in the preview release?

Reply:
No, all these properties are available in the last GA release. This test case is testing that [session-id is passed or not](https://github.com/Azure/azure-sdk-for-net/blob/6cb4b4360c6ce684435bcb2d888617267d10d8bf/sdk/search/Azure.Search.Documents/tests/DocumentOperations/SearchTests.cs#L1019).

For more details [here](https://github.com/Azure/azure-sdk-for-net/issues/27549) is an issue and [fix](https://github.com/Azure/azure-sdk-for-net/pull/27582) for this.

Reply:
Got it - thanks!